README.md

```diff
@@ -26,7 +26,7 @@ Put your credentials to the fog configuration file:
 default:
   scaleway_organization: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx # Your organization ID
   scaleway_token: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx        # Your token
-  scaleway_region: par1                                       # par1 or ams1
+  scaleway_region: fr-par-1                                   # availability zone
```

Comment:
Do you think we should change the name of the option to `scaleway_zone`?

https://developers.scaleway.com/en/quickstart/#region-and-zone